deployments/charts/kuma/values.yaml

```diff
@@ -196,6 +196,17 @@ kumactl:
     # -- The kumactl image repository
     repository: kumactl
 
+kubectl:
+  # bitnami maintains an image for all k8s versions */ } }
+  # see: https://hub.docker.com/r/bitnami/kubectl */ } }
+  image:
+    # -- The kubectl image registry
+    registry: bitnami
+    # -- The kubectl image repository
+    repository: kubectl
+    # -- The kubectl image tag
+    tag: latest
```

Comment:
This seems like it will be difficult for users to maintain and update with their cluster version – what's the reason not to infer the default from `.Capabilities.KubeVersion`?